conbench/entities/_comparator.py

```diff
@@ -187,8 +188,8 @@ def formatted(self):
             "regression": self.regression,
             "improvement": self.improvement,
             "threshold_z": fmt(self.threshold_z),
```

Comment:
Should we also send the threshold_z as a float too?

Reply:
Fixed. I checked that this change will not affect UI.